meson.build

```diff
@@ -0,0 +1,284 @@
+project('opus', 'c',
+  version: '1.3.1',
+  default_options : [ 'buildtype=debugoptimized' ]
```

Comment:
This should not be here. You should also add a license entry.

Reply:
added license entry

When I don't specify the buildtype, I get this:
```
../src/opus_decoder.c:37:10: note: ‘#pragma message: You appear to be compiling without optimization, if so opus will be very slow.’
   37 | # pragma message "You appear to be compiling without optimization, if so opus will be very slow."
      |          ^~~~~~~
```
 So, I wonder if not switching to optimized build by default is a good idea in this case...

Reply:
https://mesonbuild.com/Builtin-options.html#core-options

> For example, `-Dbuildtype=debugoptimized` is the same as `-Ddebug=true -Doptimization=2` and vice-versa.

Reply:
If you believe there is a compelling case to set `default_options: ['optimization=2']` then it should at least be commented so no one forgets why this is being done.

Reply:
Added a comment about compile-time warning when optimizations are disabled; changed `default_options` to `'optimization=2'`, leaving debug setting unmodified.